en/docs/learn/try-authorization-code-grant.md

```diff
@@ -19,9 +19,9 @@ without PKCE.
     Attacks](../../administer/mitigating-authorization-code-interception-attacks)
     to configure PKCE for an OAuth application.
 
-#### **Running the application (without PKCE)**
+## Running the application (without PKCE)
 
-1.  Visit the URL <http://wso2is.local:8080/playground2/oauth2.jsp> to
+1.  Visit the URL <http://localhost:8080/playground2/oauth2.jsp> to
```

Comment:
should this be `localhost.com` instead of `localhost`. IIRC, there was an issue with using `localhost` right? Pls check

Reply:
localhost worked fine.